cmd/rebasehelpers/commitchecker/validate.go

```diff
@@ -28,11 +28,11 @@ UPSTREAM commits are validated against the following regular expression:
 
 Examples of valid summaries:
 
-  UPSTREAM: 12345: An origin change
-  UPSTREAM: docker/docker: 12345: A docker change
+  UPSTREAM: 12345: A kube fix
+  UPSTREAM: github.com/coreos/etcd: 12345: An etcd fix
```

Comment:
make sure there are at least two segments, and that all the changed files under godeps match "Godeps/_workspace/src/" ... optional single segment (lets us skip gopkg.in or github.com, etc) ... "segment1/segment2/" ...?